### DISCUSSION TITLE
Small Suggestion: Nearest Neighbor / Hard Edges option for Selections

### DISCUSSION THREAD

**Discussion by rabbitwithworms** [keiwando/toonsquid#424] [Ideas]
Hey! Congrats on the new update! I'm really loving it right now, playing with all the new options and stuff.

It's been a while so correct me if I posted in the wrong category, but It would be nice to have a Nearest Neighbor / Hard Edges option for selections, like for example if you wanted to make a cool glitchy effect for flipnote like animations or pixel art stuff. Its pretty straightforward.
![IMG_4494](https://github.com/keiwando/toonsquid/assets/132721101/3f70d86d-6f06-45b3-ad1f-8ee26fceef49)
Transforms have Nearest Neighbor effects, so why not Selections as well?

**Comment by DinoAwakening** [comment]
https://github.com/keiwando/toonsquid/assets/120270913/10be99f9-a38f-4d14-bc47-63640908e125

I don’t think this is what you were looking for, but is the best ToonSquid can do with pixel art rn :D(?

**Reply by rabbitwithworms** [reply]
Yeah, something kinda like that, except with the selection tool, as like in the example I posted. As of this update there is no way to remove the feathering around the edges of the selection

**Comment by keiwando** [comment]
Thanks for the suggestion!
